Add forum & chat (#30)
tikv/website: content/blog/tikv-community-connections.md

<!--more-->

These spaces are designed so our community can discuss and share together. During our planning we realized that a synchronous space such as Slack or Discord offers  a good place for ephemeral *conversations*, *discussions*, and *troubleshooting*, while spaces such as forums or Discourse offer a semi-permanent spaces for sharing *articles*, *stories*, and *guides*.

This space isn't just for TiKV! You'll also find topics for some of TiKVs dependencies as well: [`raft-rs`](https://github.com/pingcap/raft-rs), [`gRPC-rs`](https://github.com/pingcap/grpc-rs/), [`rust-prometheus`](https://github.com/pingcap/rust-prometheus/), and [`fail-rs`](https://github.com/pingcap/fail-rs).

As with all TiKV project spaces, please follow the [code of conduct](https://github.com/tikv/tikv/blob/master/CODE_OF_CONDUCT.md) with us so we can foster a vibrant, accepting, worldwide community.

> The spaces is still quite young and in continuous evolution, but we'd love to have you join us and share.

Reviewer: ```suggestion
The spaces are still quite young and in continuous evolution, but we'd love to have you join us and share.
```

Author: Doh!

---
title: TiKV Community Connections
date: 2019-04-03
author: Ana Hobden, TiKV Ecosystem Lead
---

> A few months ago at [TiDB DevCon](https://pingcap.com/community-cn/devcon2019/) we talked about how we'd be opening new TiKV community forums & a community chat.

Author: Since the next paragraph is short says roughly the same, I combined them!

![image](https://user-images.githubusercontent.com/130903/55585420-73ac2980-56db-11e9-8579-e491e898d296.png)

<!--more-->

We're very happy to say you can now join us at [forum.tikv.org](http://forum.tikv.org/) for forums, and the [tikv-wg slack](https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LTgzZDQ3NzZlNDkzMGIyYjU1MTA0NzIwMjFjODFiZjA0YjFmYmQyOTZiNzNkNzg1N2U1MDdlZTIxNTU5NWNhNjk) for chat.

> We're happy to have these spaces for our community to discuss and share. During our planning we realized that a syncronous space such as Slack or Discord offer a good place for ephemeral *conversation*, *discussion*, and *troubleshooting*, while spaces such as forums or Discourse offer a semi-permanent space for sharing *articles*, *stories*, and *guides*.

Reviewer: Already stressed that we are happy :)
```suggestion
These spaces are designed for our community to discuss and share. During our planning we realized that a synchronous space such as Slack or Discord offers a good place for ephemeral *conversations*, *discussions*, and *troubleshooting*, while spaces such as forums or Discourse offer semi-permanent places for sharing *articles*, *stories*, and *guides*.
```

Author: Aww here I was extra happy

> Please respect our [code of conduct](https://github.com/tikv/tikv/blob/master/CODE_OF_CONDUCT.md) in these spaces.

Author: Moved and softened the tone dramatically
